### ISSUE TITLE
A couple questions about the implementation of smooth scrolling and quadraticCurveTo

### ISSUE THREAD

**Issue by xtianus79** [pixijs/graphics-smooth#38]
I am wondering if it is possible to better achieve biezing by adding tension so I can decrease the amount in general. Right now with this way I don't have a way to control it. That is something I need to work out.  Any suggestions would be great nevertheless.

A question I have about the "animation" is that it kind of goes from point to appearing point. Not really a crawl animation that is from 1 point to 1 point. I think that would look better. Is there a way to do that? I realize that if I probably do that it would decrease performance significantly. 

The other question I have is about the fps control. If I have data points coming in at, let's say 100 data points a second, so like every 20 - 40 ms what happens with the data that is "lagging" behind the FPS control? Just wondering that. 

I've included the pen to illustrate the point(s) :)O get it?  
CodePen

https://codepen.io/xtianus/pen/OJweLGG

**Comment by ivanpopelyshev** [comment]
Yes, its all possible, if you look at how plugin works and study it. I suggest you ot look what bufferSubData is and techniques on how to upload only changed part of buffer (pixi-candles ref). Or just create multiple graphics objects instead of one.

I have no time to explain further, sorry, you have to wait for someone else to study the same problem )

**Comment by xtianus79** [comment]
I'm going to have to pay someone for this. I will study it more. I am sure I can get the data part but I think I need someone to take it all the way. Thanks for the great plugin though.
